### ISSUE TITLE
Order of filter types after search is suboptimal, op

### ISSUE THREAD

**Issue by simon-clematide** [impresso/impresso-frontend#903]
The order of filter types make the filters from semantic enrichment (entities/topics) come last. Which hides one of our assets down in the scrolling area.
From my perspective I would want to have them offered a bit higher in the list. And maybe collapse all of them in the beginning so that a user gets an overview on the filters. 

![image](https://user-images.githubusercontent.com/3041168/83728944-404e8d00-a647-11ea-8a73-c6a75c38ec71.png)

**Comment by danieleguido** [comment]
eheh ok; I'll try to collapse all filters by default

**Comment by simon-clematide** [comment]
Yes, and after that the stateful app probably remembers (and always shows the active filters).  As soon as we will have more newspapers, the collapsing is ever more important.
